A32NX/html_ui/Pages/A32NX_Core/FMGC/A32NX_FlightPhaseManager.js

```diff
@@ -142,13 +142,23 @@ class A32NX_FlightPhase_PreFlight {
     }
 
     check(_deltaTime, _fmc) {
+        const ra = Simplane.getAltitudeAboveGround();
+
         return this.takeoffConfirmation.write(
+            Math.round(ra / 100) !== Math.round(Simplane.getAltitude() / 100) && ra > 1.5 ||
```

Comment:
Maybe let's add a comment here on exactly why you're doing this, or move it to a helper function. It's pretty obscure if you find this and don't know about the background.

Reply:
absolutely.